types/scrivito/index.d.ts

```diff
@@ -464,7 +476,7 @@ export class Team {
 export function canWrite(): boolean;
 export function configure(options: ConfigOptions): void;
 export function configureContentBrowser(options: any): void;
-export function connect(component: React.ComponentClass | React.FunctionComponent): void;
+export function connect<C>(component: C): C;
```

Comment:
Was there supposed to be a constraint on `C`?

Reply:
Yea, on React Components. Thanks, I missed this

Reply:
Hi @amcasey
thanks for your feedback. We added the constraints in ~f142068.
`export function connect<C extends React.ComponentType<any>>(component: C): C;`